cloudvolume/cloudvolume.py

```diff
@@ -1055,10 +1056,19 @@ def download_point(self, pt, size=256, mip=None):
 
     pt = self.point_to_mip(pt, mip=0, to_mip=mip)
     bbox = Bbox(pt - size2, pt + size2)
+
+    if allow_cropped:
+      bbox = Bbox.clamp(bbox, self.mip_bounds(mip))
     
     saved_mip = self.mip 
-    self.mip = mip 
-    img = self[bbox]
+    self.mip = mip
+    try:
+      img = self[bbox]
+    except ValueError:
+      self.mip = saved_mip
+      print(traceback.format_exc())
+      raise ValueError(
```

Comment:
Maybe we should make a new exception like `OutOfBoundsError`. Upside: more clear downside: requires an import from cloudvolume.exceptions to handle.

Reply:
I like it